.gitignore

```diff
@@ -28,6 +28,14 @@ secret/
 pyoidc*
 foo.*
 
+# Windows testing stuff
```

Comment:
Wow this file is a mess... This should really contain only files that are regularly generated by python itself. All the other files generated by IDE or OS should be a part of user `.gitignore`

I guess that `.cache` and `.coverage` can remain here but the rest should go to your personal `.gitignore`.

Reply:
Agreed. Will do.